CRM/CiviAwards/Service/AwardImport.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+use CRM_CiviAwards_Helper_CaseTypeCategory as CaseTypeCategory;
+
+/**
+ * Utility class for importing Awards using CSV importer extension.
+ */
+class CRM_CiviAwards_Service_AwardImport {
+
+  /**
+   * Create an Award and the related entities.
+   *
+   * @param array $params
+   *   Information for creating the award and related entities.
+   */
+  public function create(array $params) {
+    $tx = new CRM_Core_Transaction();
+
+    $params['case_type_category'] = $this->getCaseCategoryValueForAwards();
```

Comment:
We can simply do this: `$params['case_type_category'] = 'awards';` Civi will do the convert to the option value behind the scene. You can try this on the API explorer to see what I mean.

Reply:
Ok! Done: https://github.com/compucorp/uk.co.compucorp.civiawards/pull/164/commits/35bc299fcf06151a2e88b22dd80cd5ea14683c24